stream-access-control/access-control-core/src/main/java/com/backbase/stream/service/AccessGroupService.java

```diff
@@ -1354,14 +1355,16 @@ private AssignedPermission mapAssignedPermission(
         ServiceAgreement serviceAgreement,
         List<BusinessFunctionGroup> existingBusinessGroups) {
 
-        log.warn("Start Job Role updating: {}", existingBusinessGroups);
+        log.info("Start Job Role updating: {}",
+            existingBusinessGroups.stream().map(BusinessFunctionGroup::getName).collect(Collectors.toList()));
 
         streamTask.info(FUNCTION_GROUP, SETUP_FUNCTION_GROUP, "update", serviceAgreement.getExternalId(), null,
             "Update business function groups for service agreement: %s",
             serviceAgreement.getName());
 
         List<PresentationFunctionGroupPutRequestBody> presentationFunctionGroupPutRequestBody =
             existingBusinessGroups.stream()
+                .filter(businessFunctionGroup -> !businessFunctionGroup.getType().equals(TypeEnum.TEMPLATE))
```

Comment:
@Aboruhen could you if confirm this `updateJobRole` method isn't used generically?

Reply:
@uesleilima , @Aboruhen - appreciate if you can review this at the earliest; there are few Product Led projects who are waiting for this fix and are already getting noise from their customers.